validation/gatekeeper/azureidentityformat_template.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: azureidentityformat
+spec:
+  crd:
+    spec:
+      names:
+        kind: azureidentityformat
+        listKind: azureidentityformatList
+        plural: azureidentityformat
+        singular: azureidentityformat
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package azureidentityformat
+        violation[{"msg": msg}] {
+         input.review.object.kind == "AzureIdentity"
```

Comment:
Consider using `input.review.kind.kind` instead

Reply:
thank you @ritazh , have acted on the suggestion.